tests/test_logs.py

```diff
@@ -0,0 +1,38 @@
+import os
+import pytest
+from flask import session
+
+from management_app import create_app
+
+@pytest.fixture
+def app():
+  app = create_app({
+      'TESTING': True,
+  })
+  yield app
+
+@pytest.fixture
+def client(app):
+  return app.test_client()
+
+@pytest.fixture
+def mock_login(client):
+  # Enable login auth
+  with client.session_transaction() as session:
+      session['google_id'] = 'TEST_ID_NUM'
+
+# TODO Add more test cases in the future and verify the values from database
+def test_redirect_to_exceptions(client):
+    mock_login(client)
+    response = client.get('/logs')
+    assert response.status_code == 302
```

Comment:
I'm not sure if the redirect status_code is 302 or 308. Could you check again when you run `python -m pytest`? I also feel if this test case does not matter, we can just remove it and don't need the unsure expected test result at this moment.

Reply:
Thanks for the comments. The status code here should be `308` but yeah I also feel like we can remove it for now and add it later if needed.